build.sbt

```diff
@@ -178,9 +308,23 @@ lazy val root = project
   )
   .disablePlugins(MimaPlugin)
 
+lazy val ast = project.in(file("modules/ast"))
+  .withId("sangria-ast")
+  .settings(scalacSettings ++ shellSettings)
+  .settings(
+    name := "sangria-ast",
+    description := "Scala GraphQL AST representation",
+
+    //FIXME This really shouldn't depend on any other libraries.
```

Comment:
Avoid FIXME in the code that are never fixed ;)
Better adding an issue and linking the issue here.

Reply:
Not sure what you mean. A "FIXME" will show up in the IDE list of problems. And the solution to this one is removing the `render` methods, which…will get done eventually? next, even? That's part of the discussion.

There are a lot of issues that don't have GitHub issues tracking them. And I don't even know whether they're issues, because it's hard to have that discussion with you guys, over Gitter.

Reply:
In my experience, FIXME in the code stay forever and are never tackled. That's why I try to avoid them now.
But having that as a github issue will not ensure that it will be tackle. ;) But at least it's more visible to eventual contributors.

Reply:
You realize of course that you have six-year-old GitHub issues in the project. 😏 And the build spits out 100+ warnings, with more warnings from IntelliJ. To me, a "FIXME" or "TODO" is just another tool in the drawer. It's useful when the issue doesn't warrant a full-blown GitHub issue. The IDE tracks them, in the Problems tab.

This one in particular I expect to be addressed on the way to separating out the localized string from the Sangria core code. _That_ warrants a GH issue, IMO.